### DISCUSSION TITLE
Extending introspection with custom metadata

### DISCUSSION THREAD

**Discussion by IvanGoncharov** [graphql/graphql-wg#1069] [General]
This thread intended to discuss different options to extend introspection with "custom" metadata.
It started as an overflow from July 2022 WG see, details in #1055

**Comment by IvanGoncharov** [comment]
I want to schedule a dedicated call to discuss this topic as an overflow from July 2022 WG.
As suggested here: https://youtu.be/gVCszBu_aJI?t=4886

How about July 28th, 18:00 - 20:00 UTC (same time as main WG just one week earlier)?
Also, do you think we need to set up a separate repo for this subgroup or we can have a series of less formal calls similar to "input unions" and just coordinate in this thread?

CC(everyone from July 2022 agenda) @leebyron @adron @michaelstaib @aubron @joeynenni @yaacovcr @rivantsov @robrichard @timsuchanek @twof @benjie @hwillson @n1ru4l @mmahoney @saihaj @mmatsa @AnthonyMDev @ardatan @xuorig @IvanGoncharov @doc-jones @fotoetienne

**Reply by saihaj** [reply]
@IvanGoncharov July 27 there is GraphQL-JS WG https://github.com/graphql/graphql-js-wg/blob/main/agendas/2022/2022-07-27.md so we might want push this meeting an hour before or after?

**Reply by IvanGoncharov** [reply]
@saihaj Thanks for catching this. I meant Thursday 28th, one week before WG fixed it now.
Also open to any other date and time suggestions.

**Reply by saihaj** [reply]
That works for me

**Reply by rivantsov** [reply]
works for me

**Reply by michaelstaib** [reply]
works

**Reply by AnthonyMDev** [reply]
Yeah, I’ll be there!

**Reply by hwillson** [reply]
I'm in as well - thanks for setting this up @IvanGoncharov!

**Reply by benjie** [reply]
https://www.timeanddate.com/worldclock/meetingdetails.html?year=2022&month=7&day=28&hour=18&min=0&sec=0&p1=3775&p2=110&p3=24&p4=37&p5=188&p6=496&p7=676&p8=438&p9=268&p10=234&p11=78&p12=604

I'm in.

**Reply by IvanGoncharov** [reply]
@saihaj @rivantsov @michaelstaib @AnthonyMDev @hwillson @benjie Thanks for the confirmation.
I will post the Zoom link tomorrow.

**Reply by IvanGoncharov** [reply]
@saihaj @rivantsov @michaelstaib @AnthonyMDev @hwillson @benjie 
https://zoom.us/j/98261965878?pwd=YUlwamt5OURTMUVqb1Zobjh0UFhxUT09
Password: metadata

Big thanks to @benjie for creating it for us 🙇

**Reply by yaacovCR** [reply]
Any notes from this call?

**Reply by hwillson** [reply]
👋 @yaacovCR - meeting notes [here](https://docs.google.com/document/d/1b9fVkHm_QVIo92PLW32XoVKpwnKmzSX9xQGn0vVUVhA/edit#) and a prelim RFC [here](https://github.com/graphql/graphql-wg/pull/1081).

**Comment by yaacovCR** [comment]
I second the notion discussed in the linked meeting notes to refer to this new feature as “annotations” rather than “metadata.”

Metadata to me is a term limited to data about data, while we — I believe — are trying to annotate GraphQL elements rather than actual data.
